api/src/main/java/com/epam/pipeline/entity/billing/RunBilling.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.pipeline.entity.billing;
+
+import lombok.Builder;
+import lombok.Value;
+
+import java.time.LocalDateTime;
+
+@Value
+@Builder
+public class RunBilling {
```

Comment:
I think it would be useful to add `computeType` and `billingCenter` fields to the report fields

Reply:
Added.